src/main/java/address/controller/MainController.java

```diff
@@ -44,6 +44,9 @@
  * The controller that creates the other controllers
  */
 public class MainController extends UiController{
+    public static final String STAGE_TITLE_TAG_SELECTION = "Tag Selection";
+    public static final String STAGE_TITLE_MANAGE_TAG_DIALOG = "List of Tags";
+    public static final String STAGE_TITLE_NEW_TAG = "New Tag";
```

Comment:
Let's name dialogs consistently. Perhaps by the action use is performing? e.g. `Tagging a Person`, `Adding a New Tag`, `Listing All Tags` etc.